packages/contracts/test/router/v6/sdk.test.ts

```diff
@@ -44,10 +46,300 @@ describe("[ReservoirV6_0_0] Filling listings and bids via the SDK", () => {
     seaportApprovalOrderZone = (await ethers
       .getContractFactory("SeaportApprovalOrderZone", deployer)
       .then((factory) => factory.deploy())) as any;
+
+    permit2Module = (await ethers
+        .getContractFactory("Permit2Module", deployer)
+        .then((factory) =>
+          factory.deploy(deployer.address)
+        )) as any;
   });
 
   afterEach(reset);
 
+  it("Fill multiple cross-currency Seaport listings with usdc", async () => {
+
+    async function swapUSDC() {
+      const router2 = new Sdk.RouterV6.Router(chainId, ethers.provider);
+      const swapExecutions = [
+        // 1. Swap ETH for USDC on UniswapV3
+        {
+          module: router2.contracts.uniswapV3Module.address,
+          data: router2.contracts.uniswapV3Module.interface.encodeFunctionData("ethToExactOutput", [
+            {
+              tokenIn: Sdk.Common.Addresses.Weth[chainId],
+              tokenOut: Sdk.Common.Addresses.Usdc[chainId],
+              fee: 500,
+              recipient: dan.address,
+              amountOut: parseUnits("10000", 6),
+              amountInMaximum: parseEther("10"),
+              sqrtPriceLimitX96: 0,
+            },
+            dan.address,
+          ]),
+          // Anything on top should be refunded
+          value: parseEther("10"),
+        },
+      ];
+
+      // Swap to erc20
+      try {
```

Comment:
The swap can't fail (if it fails something is wrong with the test setup) so let's remove the `try / catch` around it.